pkgs/development/interpreters/trealla/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, stdenv, fetchFromGitHub, readline, openssl }:
+
+stdenv.mkDerivation rec {
+  pname = "trealla";
+  version = "1.7.35";
+
+  src = fetchFromGitHub {
+    owner = "infradig";
+    repo = "trealla";
+    rev = "v${version}";
+    sha256 = "sha256-e50IL6MW8PHTCqKZUlPwreUtm0cYhXuULenxkbxdZ0I=";
+  };
+
+  postPatch = ''
+    substituteInPlace Makefile \
+      --replace '-I/usr/local/include' "" \
+      --replace '-I/usr/local/lib' ""
+  '';
+
+  makeFlags = [ "GIT_VERSION=\"v${version}\"" ] ++ lib.optional stdenv.isDarwin "NOLDLIBS=1";
```

Comment:
Are you sure that this "setting" of `GIT_VERSION` does what you expect from it?

Reply:
The line in https://github.com/infradig/trealla/blob/v1.7.35/Makefile#L1 sould be ?= IIRC. Good catch.

Reply:
nah, I looked at the GNU make manual, and gmake allows overriding variables assigned using `:=` on the command line. I don't know if any other make may be used. https://www.gnu.org/software/make/manual/html_node/Overriding.html#Overriding